tests/core/method-class/test_method.py

```diff
@@ -69,7 +72,7 @@ def test_get_formatters_non_falsy_config_retrieval():
     first_formatter = (method.request_formatters(method_name).first,)
     all_other_formatters = method.request_formatters(method_name).funcs
     assert len(first_formatter + all_other_formatters) == 2
-    # assert method.request_formatters('eth_nonmatching') == 'nonmatch'
+    assert (method.request_formatters('eth_getBalance').first,) == first_formatter
```

Comment:
I just tried to test _something_ here because this test wasn't doing much in the state it was in.